### ISSUE TITLE
Install issue: go get is deprecated. Need to use go install

### ISSUE THREAD

**Issue by jmorris644** [readium/readium-lcp-server#261]
in the current version of go (and maybe a couple of versions back), the command "go get" has been deprecated. The error message states that "go install" needs to be used instead. So the install is not working properly.

I am a nodejs guy so I do not know where to go to temporarily fix this.

**Comment by llemeurfr** [comment]
It is true that starting in Go 1.17, installing executables with go get is deprecated. go install may be used instead. 
I'll update the install documentation asap to reflect this.

**Comment by mickael-menu** [comment]
This worked for me:

```sh
$ go install github.com/readium/readium-lcp-server/...@latest
```

**Comment by llemeurfr** [comment]
Fixed via b17cfc5de86814e7048bbd5fe05231da943a372c
